### ISSUE TITLE
chat empty after being offline

### ISSUE THREAD

**Issue by Danfro** [holochain-apps/relay#174]
v0.7.0-beta-rc.3

I started testing the app yesterday. I wrote some messages. Over night I turned the phone off(line) and closed the app. When opening the test group chat, it eventually updated the member list, but there are no (zero) messages there. The chat comes up entirely empty.

Waiting for 15 minutes now without anything from last night showing up.

I wrote some more messages today. After closing the app they where gone too.

**Comment by Danfro** [comment]
0.7.0-beta same

**Comment by mattyg** [comment]
Hi @Danfro -- thanks for the bug report. Just to confirm these were your own **sent** messages that no longer appear in the group chat after closing and reopening the app?

**Comment by Danfro** [comment]
@mattyg no, zero means zero in this case. There are no messages at all. Not my sent ones, no received ones, simple no messages at all.

In case it matters, running the app on a Xperia X with Lineage OS 20, no GooglePlayServices/MicroG running.

**Comment by Danfro** [comment]
Hmm. I just checked with other users. They did send messages last night, but I didn't receive anything. Looks like I am "blind".

But if I close the app and reopen it, the number of members gets updated. So there seems to be at least some interaction.

**Comment by Danfro** [comment]
I do receive some messages. So might also depend on who is writing and how many members are online maybe?

**Comment by wurzer** [comment]
Outdated. Needs to be retested
